src/sidebar/tabcenter.css

```diff
@@ -630,7 +632,7 @@ body[platform="win"] #searchbox-input:placeholder-shown {
 }
 
 .tab.discarded .tab-title {
-    color: var(--secondary-text-color);
+    opacity: 0.5;
```

Comment:
Note (unrelated): We should try to do something similar (`.7`) for the title wrapper when a tab is not active.